lib/shared/theme/color_scheme.dart

```diff
@@ -1,68 +1,31 @@
 import 'package:flutter/material.dart';
 
 const lightColorScheme = ColorScheme(
-  brightness: Brightness.light,
-  primary: Color(0xff212121),
-  onPrimary: Color(0xffFFEC19),
-  primaryContainer: Color(0xff484848),
-  onPrimaryContainer: Color(0xffFFEC19),
-  secondary: Color(0xff7E57C2),
-  onSecondary: Color(0xffffffff),
-  secondaryContainer: Color(0xffF3E5F5),
-  onSecondaryContainer: Color(0xff000000),
-  tertiary: Color(0xff7E57C2),
-  onTertiary: Color(0xffffffff),
-  tertiaryContainer: Color(0xffD1C4E9),
-  onTertiaryContainer: Color(0xff000000),
-  error: Color(0xffD32F2F),
-  onError: Color(0xffffffff),
-  errorContainer: Color(0xffEF9A9A),
-  onErrorContainer: Color(0xff000000),
-  background: Color(0xffFFFBEA),
-  onBackground: Color(0xff000000),
-  surface: Color(0xffFFFBEA),
-  onSurface: Color(0xff000000),
-  surfaceVariant: Color(0xffEDE7F6),
-  onSurfaceVariant: Color(0xff000000),
-  outline: Color(0xffBDBDBD),
-  inverseSurface: Color(0xff424242),
-  onInverseSurface: Color(0xffFFEC19),
-  inversePrimary: Color(0xffFFEC19),
-  shadow: Color(0xff000000),
-  surfaceTint: Color(0xff212121),
-  outlineVariant: Color(0xff8D8D8D),
-);
+    brightness: Brightness.light,
+    primary: Color(0xff161616),
+    onPrimary: Color(0xffFFEC19),
+    secondary: Color(0xff9A1AFF),
```

Comment:
how come `lightColorScheme` uses purple in `secondary` and `onSurfaceVariant`? not enough ✨color✨?

Reply:
think for now we should stick with black + white + yellow in the light theme (meaning `secondary` and `surfaceVariant` would be some variation of black/dark gray) but i have a feeling the whole color theme is something we'll be coming back to throughout the year!